src/components/molecules/PinInput/PinInput.scss

```diff
@@ -0,0 +1,74 @@
+@import '../../../assets/styles/fonts.scss';
```

Comment:
@PRASHANTK22111993 Do we need this import, we can remove this right?

Reply:
@Rajmohanrk Added the font-family Attribute we needed that.